pypesto/objective/amici.py

```diff
@@ -144,18 +144,28 @@ def __init__(self,
                 amici_model, len(edatas))
         self.parameter_mapping = parameter_mapping
 
-        # preallocate guesses, construct a dict for every edata for which we
-        # need to do preequilibration
-        if self.guess_steadystate:
-            if self.amici_model.ncl() > 0:
-                raise ValueError('Steadystate prediction is not supported for '
-                                 'models with conservation laws!')
-
-            if self.amici_model.getSteadyStateSensitivityMode() == \
-                    amici.SteadyStateSensitivityMode_simulationFSA:
-                raise ValueError('Steadystate guesses cannot be enabled when'
-                                 ' `simulationFSA` as '
+        # If supported, enable `guess_steadystate` by default. If not
+        #  supported, disable by default. If requested but unsupported, raise.
+        if self.guess_steadystate is not False and self.amici_model.ncl() > 0:
```

Comment:
one could apply the change from  #561 here...

Reply:
yes, done